conrad-blucher-institute/semaphore
# 156 update mlp op dspec and run coldstunning

Joy: Originally assigned to me. I updated a part of the DSPEC. Beto took over to fix bugs with NDFD that were causing issues with running the cold stunning model.

## Beto: How to test:
- Ensure correct .env variables are set. See `env.dist`
- Run `docker compose build` and `docker compose up` (run `docker compose up -d` to run in the background)
- SSH into semaphore container: `docker exec -it semaphore-core bash`
- Navigate and run commands from inside the container:
     -  `python3 tools/init_db.py` to init the DB
     -  `python3 src/semaphoreRunner.py -d MLP-OP.json` to run the model

## Important Note
- All inputs must be cleared from database if running for the same referenceTime. (Either delete the DB volume on Docker Desktop and init_db.py again. Or TRUNCATE the inputs table in pgAdmin.) This is because there is a bug where seriesProvider will return both the data ingestion data and the DB data even if it is the same data.
